### PR TITLE
Add servo.proto documentation

### PR DESCRIPTION
**Pull request by brentru**

### CODE REVIEW

File: proto/wippersnapper/servo/v1/servo.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+# servo.proto
+
+  This file details the WipperSnapper messaging API for interfacing with servo output components.
+
+## WipperSnapper Components
+
+  
+
+The following WipperSnapper components utilize `pwm.proto`:
```
Comment (lorennorman) [inline proto/wippersnapper/servo/v1/servo.md:10]: old reference: `pwm.proto`

File: proto/wippersnapper/servo/v1/servo.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+  
+
+### Delete: Servo
+
+```mermaid
+sequenceDiagram
+autonumber
+IO->>Device: ServoDetachRequest
```
Comment (lorennorman) [inline proto/wippersnapper/servo/v1/servo.md:63]: isn't there a pin required to know which servo to delete?

**Review by lorennorman — APPROVED** [review]
couple changes but looks great

**Comment by brentru** [comment]
Changes addressed in https://github.com/adafruit/Wippersnapper_Protobuf/pull/120/commits/c2394e6bb8f25e6c9c4713c20aa43fb2b4242704, seeing as this is approved, I'll merge.

**Pull request merged**